### PR TITLE
oauth config: load oauth client secrent from file

### PR DESCRIPTION
**Pull request by jkralik**

### CODE REVIEW

File: docs/guide/deployment/resource-aggregate.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ clients:
         certFile: "/data/certs/http.crt"
     oauth:
       clientID: "412dsFf53Sj6$"
-      clientSecret: "235Jgdf65jsd4Shls"
+      clientSecretFile: "235Jgdf65jsd4Shls"
```
Comment (Danielius1922) [inline docs/guide/deployment/resource-aggregate.md]: Not a good example file path.

EDIT: but In my PR I've removed oauth so it doesn't matter

Comment (jkralik) [reply docs/guide/deployment/resource-aggregate.md]: But i fixed :D

**Review by Danielius1922 — APPROVED** [review]

File: bundle/run.sh [hunk 2]
```diff
@@ -56,6 +57,8 @@ export SECRETS_DIRECTORY=/data/secrets
 export OAUTH_ID_TOKEN_KEY_PATH=${OAUTH_KEYS_PATH}/id-token.pem
 export OAUTH_ACCESS_TOKEN_KEY_PATH=${OAUTH_KEYS_PATH}/access-token.pem
 
+export OAUTH_DEVICE_SECRET_PATH=${OAUTH_SECRETS_PATH}/device.secret
```
Comment (ondrejtomcik) [inline bundle/run.sh:60]: please update this doc https://plgd.dev/tutorials/external-oauth-server/

File: docs/guide/deployment/cloud2cloud-connector.md [hunk 3]
```diff
@@ -166,7 +166,7 @@ docker run -d --network=host \
 | `apis.http.pullDevices.disabled` | bool | `Disable get devices via pull for all clouds` | `true` |
 | `apis.http.pullDevices.interval` | string | `"Time interval between pulls` | `"5s"` |
 | `apis.http.clientID` | string | `Client ID to exchange an authorization code for an access token.` | `""` |
-| `apis.http.clientSecret` | string | `Client secret to exchange an authorization code for an access token.` | `""` |
+| `apis.http.clientSecretFile` | string | `File path to client secret to exchange an authorization code for an access token.` | `""` |
```
Comment (ondrejtomcik) [inline docs/guide/deployment/cloud2cloud-connector.md]: File path to client secret required to ex.

**Review by ondrejtomcik — APPROVED** [review]

**Comment by ondrejtomcik** [comment]
@rafajpet ping

File: bundle/run.sh (hunk 2 above)
Comment (jkralik) [reply bundle/run.sh:60]: I think @Danielius1922 will update it because he removes lots of ENV vars.

**Pull request merged**